Underscore/USArrayWrapper.m

```diff
@@ -168,8 +168,9 @@ - (USArrayWrapper *)shuffle
 {
     NSMutableArray *result = [self.array mutableCopy];
 
+    NSInteger max = result.count;
```

Comment:
No problem. Although my reading suggests that the assert might still be overkill.  I don't thin we'll have iOS/MacOS hardware that will support that many elements stored in an array anytime soon.